custom_model_runner/requirements.txt

```diff
@@ -15,4 +15,4 @@ texttable
 py4j~=0.10.9.0
 pyarrow==2.0.0
 Pillow==8.2.0
-julia==0.5.6
\ No newline at end of file
+julia==0.5.6
```

Comment:
Not sure why this has a diff change in the Github UI, I did not do any change to the julia constraint version